geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/text/TextTools.scala

```diff
@@ -33,4 +35,30 @@ object TextTools {
       new String(Array.fill(length)(c))
     }
   }
+
+  /**
+    * Builds a natural word list, e.g. 'foo, bar, baz and blu'
```

Comment:
Does this code use the Oxford comma or not?

Seem it would output 'foo, bar, baz, and blu'...;)

Reply:
nope, no oxford comma here :)

Reply:
Ah, I was reading da code too quickly.  

That said, I'm a fan of more commas...